src/types/swiper.d.ts

```diff
@@ -0,0 +1,10 @@
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'swiper-container': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
+      'swiper-slide': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
```

Comment:
Waw their typescript support sucks.

You can extend global to improve event types too
```ts
  interface ElementEventMap {
    'slidechange': SlideChangeEvent
  }
```